pom.xml

```diff
@@ -78,6 +78,9 @@
         <!-- Mixed Reality -->
         <module>sdk/mixedreality/azure-mixedreality-authentication</module>
 
+        <!-- DigitalTwins Model Parser -->
+        <module>sdk/modelparser/azure-digitaltwins-parser</module>
```

Comment:
digital twins parser sounds like a thing that parses digital twins, not digital twins models. I know you aren't defining the name, rather going with the precedent from other languages, but I think it is a worthwhile question to bring up. Thoughts?

Reply:
I absolutely agree, that's why I left it for debate, I think specifying the `model` is necessary here.

Reply:
I will start a conversation about the namespace to the team and the PMs.